### ISSUE TITLE
Incorrect input sequencing shape during prediction with batchsize>1

### ISSUE THREAD

**Issue by eternalding** [rschwess/deepC#2]
Greetings,

I trained DeepC model (tensorflow2.1plus_compatibility_version) w/ batchsize of 2 following your tutorials.

However, when I was trying to do a prediction, I got an error message "Invalid argument: Input to reshape is a tensor with 100500 values, but the requested shape has 201000" from the reshape layer.

I think the main reason for this problem is the shape of seqs_placeholder in predict function.

Using np.expand_dims will create a new dimension with the size of 1, which is not capable for models with batchsize>1.

Could you check out the code and see if my assumption is correct? Thank you.

Wenny

**Comment by rschwess** [comment]
Hi Wenny, Indeed that np.expand_dims call is the problem. 
All it's doing is taking the first entry along the first batch dimension only to then expand the dimension again by 1 only. Works for batch sizes of 1 but not for everything else. 

Changing line 143 in 'run_deploy_shape_deepCregr.py' from

`seqs_placeholder: np.expand_dims(seqs[test_batch_range][0], axis=0),`

to

`seqs_placeholder: seqs[test_batch_range],`

fixes the dimension problem for both batch size 1 and 2 models on my end. Could you give that a try?

Will run a few more tests before changing it on the master version.

Practically I only worked with a batch size of 1 due to GPU memory restrictions so I wouldn't be surprised if you stumble over another batch size bug, please let me know when you do!

**Comment by eternalding** [comment]
Thank you for your instant reply. 
I've tried your suggestions, looks good to me.

Wenny

**Comment by rschwess** [comment]
Fixed now for all prediction scripts
